next.config.js

```diff
@@ -4,4 +4,56 @@ module.exports = {
   future: {
     webpack5: true,
   },
+  async redirects() {
+    let redirects = maintenanceMode();
+    return redirects.filter(Boolean);
+  },
 };
+
+function maintenanceMode() {
+  let maintenance_array = [];
+
+  if (
+    process.env.NEXT_PUBLIC_MAINTENANCE_MODE &&
+    process.env.NEXT_PUBLIC_MAINTENANCE_MODE === '1'
+  ) {
+    let pages = [
+      '/access-denied',
+      '/forms-in-progress',
+      '/login',
+      '/logout',
+      '/index',
+      '/search',
+      '/my-records',
+      '/workers',
+      '/workers/:slug*',
+      '/api/:slug*',
+      '/people',
+      '/people/:slug*',
+      '/submissions:slug*',
```

Comment:
Is this missing a slash? 🕵️ 

```suggestion
      '/submissions/:slug*',
```